### ISSUE TITLE
MapAlignerPoseClustering on Galaxy

### ISSUE THREAD

**Issue by trachtok** [galaxyproteomics/tools-galaxyp#241]
Hello,

I would like to use function MapAlignerPoseClustering from OpenMS in my Galaxy workflow for mass spectrometry data. However, this function is not part of the OpenMS wrapper for Galaxy. What I found on your GitHub page is: "These tools have multiple outputs (number of inputs = number of outputs) which is not yet supported in by the automatic conversion step". Is there really no way of getting this function into Galaxy? I was able to write wrappers for some of my tools, but I have no luck with MapAlignerPoseClustering as it should take a dynamic number of inputs and give the same number of output files, which is a thing I was not able to figure out. Do you have any ideas how to do this or it is impossible for Galaxy?

Thank you very much,
Karolina Trachtova

**Comment by bgruening** [comment]
Hi @trachtok - I have added https://github.com/galaxyproteomics/tools-galaxyp/pull/242

By consistence I was working on this tool, since a lab at our university was in the need of this tool as well. Before I merge it, I wait currently for some testers to confirm that it works.

> I was able to write wrappers for some of my tools

Which tools are this. Do you want to contribute them to GalaxyP?

**Comment by trachtok** [comment]
That is great! Do you know when it will be available? If it's gonna be this month or later. 
The tool I wrote a wrapper for is an R package for deconvolution and some other stuff of GC-MS data. My colleague is working on it but the package is still not complete and we are still testing it. 

Thanks!

**Comment by bgruening** [comment]
@trachtok fixed in https://github.com/galaxyproteomics/tools-galaxyp/pull/242
It will be uploaded in the next hours.

Enjoy!

**Comment by trachtok** [comment]
Hi @bgruening ,

I just tried to use MapAlignerPoseClustering but I encountered a problem. I always used it with several input files at once, but I am not able to do this in Galaxy. It always runs only with one input file and that is kind of pointless as it does not have any other files against which it would do comparisons and corrections. It is probably a stupid question but how do I start MapAlignerPoseClustering on Galaxy with several input files at once? Am I missing some button for adding another input files?

Thank you very much!
